src/main/java/org/reactome/release/downloaddirectory/GenerateGOAnnotationFile/CreateGOAFile.java

```diff
@@ -34,10 +40,11 @@ public static void execute(MySQLAdaptor dbAdaptor, String releaseNumber) throws
                 BiologicalProcessAnnotationBuilder.processBiologicalFunctions(reactionInst);
```

Comment:
I think this is out of scope of the PR, but why do the values returned by `processCellularComponents`, `processMolecularFunctions`, and `processBiologicalFunctions` never get used?

Reply:
The return values are used in the tests, but I refactored to have them be used in the main program as well.